lib/speech.js

```diff
@@ -0,0 +1,48 @@
+import {EventEmitter2} from 'eventemitter2';
+import {Message} from 'roslib';
```

Comment:
If you run `npx xo` you can check the style guide of this package, there are several small issues